src/com/phonebookservice/server/ContactParser.java

```diff
@@ -0,0 +1,14 @@
+package com.phonebookservice.server;
```

Comment:
IContactParser. 

Also please create java package for database. 

Why adding this in server package ?

Reply:
you are right